docs/specifications/cli_user_guide.md

```diff
@@ -0,0 +1,401 @@
+			"NextBookkeeper": "TACATSTXT4E6UJHY57Ccv2TNBPGjbD9hfU",
+			"PrevBlockHash": "bdbe264712d38f6d46bffdea8f6a9ab3ef73d343b043d31a0a0594758b73da1d",
+			"SigData": [
+				"016f3669a1998f2716e91093ca7946263433f92605ae9e6bf2b695145dbe24de5b38c93f8e91ae1d1f89626a4e87ac101f4344fb312c3e482b105a94f7d15f3fe9"
+			],
+			"Timestamp": 1524280951,
+			"TransactionsRoot": "a750c36fca5922a6d4fe22d6b8664faa06f37d4c79c4624a91b10198322adf0e",
+			"Version": 0
+		},
+		"Transactions": [
+			{
+				"Attributes": [],
+				"Fee": [],
+				"Hash": "a750c36fca5922a6d4fe22d6b8664faa06f37d4c79c4624a91b10198322adf0e",
+				"NetworkFee": 0,
+				"Nonce": 0,
+				"Payload": {
+					"Nonce": 1524280951763615000
+				},
+				"Sigs": [
+					{
+						"M": 1,
+						"PubKeys": [
+							"12020352f5ea426d333bd4c0f174fc93e84348513a8d3778ac0c3beaca579cf1937696"
+						],
+						"SigData": [
+							"01885e3078fe398987c5d89d650714d4e4ab9432ea5bcb98505cf4913d224920203c2f98b1ea24857e6af0f06b3bdf72d862bbe3b0989937cbe06c91775e2aa6e8"
+						]
+					}
+				],
+				"TxType": 0,
+				"Version": 0
+			}
+		]
+	},
+	"Version": "1.0.0"
+}
+
+NOTE:
+If a invalid value of block hash is given, the result will show as follow:
+{
+	"Action": "getblockbyheight",
+	"Desc": "UNKNOWN BLOCK",
+	"Error": 44003,
+	"Result": "",
+	"Version": "1.0.0"
+}
+```
+
+### Example for info transaction
+```
+$ ./ontology info tx --hash 805e8ad8de2b28884656a393143dc07fea1cc83ddd849e1bbab814c476c26bb2
+Result will show as follow: 
+{
+	"Action": "gettransaction",
+	"Desc": "SUCCESS",
+	"Error": 0,
+	"Result": {
+		"Attributes": [],
+		"Fee": [],
+		"Hash": "a750c36fca5922a6d4fe22d6b8664faa06f37d4c79c4624a91b10198322adf0e",
+		"NetworkFee": 0,
+		"Nonce": 0,
+		"Payload": {
+			"Nonce": 1524280951763615000
+		},
+		"Sigs": [
+			{
+				"M": 1,
+				"PubKeys": [
+					"12020352f5ea426d333bd4c0f174fc93e84348513a8d3778ac0c3beaca579cf1937696"
+				],
+				"SigData": [
+					"01885e3078fe398987c5d89d650714d4e4ab9432ea5bcb98505cf4913d224920203c2f98b1ea24857e6af0f06b3bdf72d862bbe3b0989937cbe06c91775e2aa6e8"
+				]
+			}
+		],
+		"TxType": 0,
+		"Version": 0
+	},
+	"Version": "1.0.0"
+}
+
+NOTE:
+If a invalid value of transaction hash is given, the result will show as follow:
+{
+	"Action": "gettransaction",
+	"Desc": "UNKNOWN TRANSACTION",
+	"Error": 44001,
+	"Result": "",
+	"Version": "1.0.0"
+}
+```
+---
+
+## <a name="cli-asset-cmd"></a> CLI Asset Command
+```
+Usage:
+    ontology asset [command options] [args]
+
+Description:
+    With this command, you can control assert through transaction.
+
+Command:
+    transfer
+        --caddr     value                 smart contract address
+        --from      value                 wallet address base58, which will transfer from
+        --to        value                 wallet address base58, which will transfer to
+        --value     value                 how much asset will be transfered
+        --password  value                 use password who transfer from
```

Comment:
use plaintext password is not safe in most cases. better input password without showing them.

anyway, if the enduser insist to use plaintext, e.g., in some shell script, this --password parameter can be applied.

please add more comment on this parameter.

Reply:
relevant pr #52